### ISSUE TITLE
ORT Scan started to fail when use a Gradle version lower than 7.2.0

### ISSUE THREAD

**Issue by abanias** [oss-review-toolkit/ort#6419]
The ort scan job started to fail with the following error: 
```
Resolving Gradle dependencies for path 'build.gradle' failed with: BuildException: Could not fetch model of type 'DependencyTreeModel' using connection to Gradle distribution 'https://services.gradle.org/distributions/gradle-6.9-bin.zip'. Caused by: LocationAwareException: Initialization script '/tmp/ort-Gradle4749349623269099033/init15891916940425317509.gradle' line: 179 Could not compile initialization script '/tmp/ort-Gradle4749349623269099033/init15891916940425317509.gradle'. Caused by: ContextualPlaceholderException: Could not compile initialization script '/tmp/ort-Gradle4749349623269099033/init15891916940425317509.gradle'. Caused by: PlaceholderException: startup failed: initialization script '/tmp/ort-Gradle4749349623269099033/init15891916940425317509.gradle': 179: unable to resolve class org.gradle.internal.composite.IncludedBuildInternal @ line 179, column 29. def state = (it as org.gradle.internal.composite.IncludedBuildInternal).getTarget() ^ 1 error
```
The problem arose because of this change in the ort project: https://github.com/oss-review-toolkit/ort/commit/6cce314c695e9e4071ef3f8a7956b2683cf2b9d5#diff-e0c99299dd78a854b2dc32308b0222986441ce3d73fea8e2efc1cabb6e39a179Ra177

Our project uses Gradle 6.9, but the getTarget() method, used in the changes above, doesn't exist in that Gradle version.
This method was added in Gradle 7.2.0.

**Comment by nnobelis** [comment]
Thanks for analysis !

Out of curiosity, does your project use the composite build feature ?
Because I don't think the error occurs if the build does not have any included builds.

@mnonnenmacher @sschuberth I guess with will have to condition the composite build logic to the Gradle version ?
Or do you see another workaround ?

**Comment by abanias** [comment]
Our project doesn't use the composite build feature.
It's not even a runtime error. It's a compilation error because Gradle 6.9 can not find the getTarget() method in the org.gradle.internal.composite.IncludedBuildInternal class.

**Comment by sschuberth** [comment]
> I guess with will have to condition the composite build logic to the Gradle version ?

Looks like so. BTW, I was wondering why our `"Is compatible with Gradle >= 2.14"` *GradleFunTest* then didn't fail here for the PR... but apparently we've disabled that test 😞 I guess we should enable it, probably with fewer versions to check, to reduce CI time.

**Comment by nnobelis** [comment]
I am preparing an emergency fix

**Comment by nnobelis** [comment]
In addition to the hotfix currently being prepared, I raised a question to the Gradle forums to get advice on a cleaner solution:
https://discuss.gradle.org/t/tapi-model-should-deliver-composite-build-informations/44776

**Comment by mnonnenmacher** [comment]
Fixed by #6431. @abanias Please re-open the issue if the fix does not work for you.
